src/components/vr_cooperation/src/vr_module.cc

```diff
@@ -137,7 +139,8 @@ functional_modules::ProcessResult VRModule::HandleMessage(
       break;
     }
     case application_manager::MessageType::kRequest: {
-      commands::Command* command = MobileCommandFactory::CreateCommand(this, msg);
+      commands::Command* command = MobileCommandFactory::CreateCommand(this,
+                                                                       msg);
```

Comment:
Sometimes you can use line which little longer then 80 characters.
Cpplint has the argument to set allowed length of line.